src/appleseed.studio/mainwindow/rendering/renderwidget.h

```diff
@@ -33,9 +33,10 @@
 #include "mainwindow/rendering/renderclipboardhandler.h"
 
 // appleseed.foundation headers.
-#include "foundation/platform/compiler.h"
 #include "foundation/image/image.h"
 #include "foundation/image/tile.h"
+#include "foundation/math/vector.h"
+#include "foundation/platform/compiler.h"
```

Comment:
I think we can safely get rid of `compiler.h`.

Reply:
ok, I just moved it so it is order alphabetical order